include/tudat/simulation/environment_setup/createRadiationPressureTargetModel.h

```diff
@@ -122,6 +122,26 @@ class CannonballRadiationPressureTargetModelSettings : public RadiationPressureT
 
     double coefficient_;
 };
+
+class PaneledRadiationPressureTargetModelSettings : public RadiationPressureTargetModelSettings
+{
+private:
+    int maximumNumberOfPixels_;
+
+public:
+    explicit PaneledRadiationPressureTargetModelSettings(
+        const std::map< std::string, std::vector< std::string > >& sourceToTargetOccultingBodies = { },
+        int maximumNumberOfPixels = 0):
```

Comment:
maximumNumberOfPixels = 0 meaning no shadowing? For internal settings, this is fine, but the user interface should be slightly different, with either:
* An interface to create a model without shadowing
* A boolean to set to False